iis-common/src/test/java/eu/dnetlib/iis/common/java/io/JsonStreamWriterTest.java

```diff
@@ -28,15 +29,14 @@ public void basicTest() throws IOException{
 		}
 		writer.close();
 		String actual = out.toString();
-		InputStream in = Thread.currentThread().getContextClassLoader()
-			.getResourceAsStream("eu/dnetlib/iis/core/java/io/document.json");
+		Reader in = new InputStreamReader(Thread.currentThread().getContextClassLoader().getResourceAsStream("eu/dnetlib/iis/common/java/io/document.json"), "UTF-8");
 		String expected = toString(in);
 		Assert.assertEquals(expected, actual);
 	}
 	
-	private static String toString(InputStream in) throws IOException{
+	private static String toString(Reader in) throws IOException{
 		StringWriter writer = new StringWriter();
 		IOUtils.copy(in, writer);
-		return writer.toString();
+		return writer.toString().replace("\r", "");
```

Comment:
Yes, there is a test error without it on windows. The problem is the different new line characters in unix and windows.